echo-core/src/main/java/com/netflix/spinnaker/echo/build/BuildInfoService.java

```diff
@@ -74,8 +74,12 @@ public BuildEvent getBuildEvent(String master, String job, int buildNumber) {
     String job = event.getContent().getProject().getName();
     int buildNumber = event.getBuildNumber();
 
+    if (StringUtils.isEmpty(propertyFile) && master.contains("travis")) {
+      propertyFile = "travis";
+    }
+    String propertyFileFinal = propertyFile;
     if (StringUtils.isNoneEmpty(master, job, propertyFile)) {
-      return retry(() -> igorService.getPropertyFile(buildNumber, propertyFile, master, job));
+      return retry(() -> igorService.getPropertyFile(buildNumber, propertyFileFinal, master, job));
```

Comment:
Is this needed with the `orca` change? It seems not idea to have an explicit travis check in the core `BuildInfoService`.

Reply:
Yes, unfortunately. This PR is for build _triggers_, the orca one is for build _stages_. The Jenkins/Travis relationship in echo is ugly (basically there's no way to differentiate them except using the name - Travis build masters are always prefixed with `travis-`).